doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -143,8 +143,14 @@ CPU reservation for host: 1 core per NUMA
     - minimum: two nodes per profile
 -	HW specifications
     -	Boot disks are dedicated with Flash technology disks
-    - In case of DPDK usage, cores (together with their sibling threads) must be reserved for DPDK Poll Mode Drivers (PMD), hugepages set to at least 1 GB, and all power settings disabled in the BIOS. The number of cores reserved for this overhead depends on the desired throughput (the number of Mpps we want to deliver).
-    
+
+- In case of DPDK usage:
+
+| Layer | Description |
+|-------------|--|
+| Cloud infrastructure | Important is placement of NICs to get to balanced IO system, and configuration of NIC features. Server BIOS and Host OS kernel command line settings are described on DPDK.org in release notes and performance reports. Disabling power settings (like Intel Turbo Boost Technology) brings stable performance results, although understanding if and when they benefit workloads and enabling them can achieve better performance results. |
+| Workload | DPDK uses core affinity along with 1G or 2M Huge Pages, NUMA settings, and DPDK Poll Mode Drivers (PMD) to get the best performance. DPDK versions xx.11 are Long-Term Support maintained stable release with back-ported bug fixes for a two-year period. |
```

Comment:
to get best performance, do we need multiple PMD threads at compute host level?

Reply:
It depends on design details explained in DPDK Programming Guide [PMD](https://doc.dpdk.org/guides/prog_guide/poll_mode_drv.html) that I would not add in RA text.